resources/todomvc/architecture-examples/angular-complex/scripts/build.js

```diff
@@ -8,11 +8,12 @@ const options = {
     callerDirectory: path.resolve(__dirname),
     sourceDirectory: path.join("..", "node_modules", "todomvc-angular", "dist"),
     title: "TodoMVC: Angular Complex DOM",
-    filesToMove: ["node_modules/big-dom-generator/dist/big-dom-generator.css", "node_modules/big-dom-generator/dist/logo.png"],
+    filesToMove: ["node_modules/big-dom-generator/dist/big-dom-generator-v2.css", "node_modules/big-dom-generator/dist/logo.png"],
```

Comment:
-v1 and -v2 suffix doesn't really explain the difference. Can we use suffix like "-isolate" so that names are more self explanatory?

Reply:
Done.